### PR TITLE
Feature: add default behaviour for clusterawsadm bootstrap iam print-policy

### PR DESCRIPTION
**Pull request by Atharva-Shinde**
**What type of PR is this?**

**What this PR does / why we need it**:

Currently clusterawsadm doesn't support execution of "clusterawsadm bootstrap iam print-policy", it needs "--document" flag to specify the policy documents we want to see.

This PR introduces the "clusterawsadm bootstrap iam print-policy" as an individual command to list all those policy documents at once.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [x] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

**Release note**:

```release-note
Update clusterawsadm to include a full list of policies when omitting `--document` parameter.
```

### CODE REVIEW

**Comment by Atharva-Shinde** [comment]
/kind feature
cc @richardcase @Ankitasw

File: cmd/clusterawsadm/cloudformation/bootstrap/iam.go [hunk 1]
```diff
@@ -71,6 +71,18 @@ func (t Template) policyFunctionMap() map[PolicyName]func() *iamv1.PolicyDocumen
...
+		policyDoc := t.GetPolicyDocFromPolicyName(name)
+		value, err := converters.IAMPolicyDocumentToJSON(*policyDoc)
+		if err != nil {
+			return err
+		}
+		fmt.Println(name, value)
+	}
+	return nil
+}
+
```
Comment (Skarlso) [inline cmd/clusterawsadm/cloudformation/bootstrap/iam.go:85]: I assume this is some kind of debug function, right? This should be removed or changed or something.

File: cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go [hunk 2]
```diff
@@ -64,6 +64,11 @@ func printPolicyCmd() *cobra.Command {
 				return err
 			}
 
+			if policyName == "" {
+				err := template.GetPolicyDocs()
+				return err
+			}
```
Comment (Skarlso) [inline cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go:70]:
```suggestion
			if policyName == "" {
				return template.GetPolicyDocs()
			}
```

File: cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go [hunk 3]
```diff
@@ -81,6 +86,11 @@ func printPolicyCmd() *cobra.Command {
 
 func getDocumentName(cmd *cobra.Command) (bootstrap.PolicyName, error) {
 	val := bootstrap.PolicyName(cmd.Flags().Lookup("document").Value.String())
+
+	if val == "" {
+		return "", nil
```
Comment (Skarlso) [inline cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go:91]: This should also probably be a validation error?

**Comment by Atharva-Shinde** [comment]
Thanks @Skarlso I have made the change

File: cmd/clusterawsadm/cloudformation/bootstrap/iam.go (hunk 1 above)
Comment (Skarlso) [reply cmd/clusterawsadm/cloudformation/bootstrap/iam.go:85]: Hey @Atharva-Shinde.

I don't see you changing this function. This function still doesn't make any sense. It's not returning anything, it doesn't set anything... And it's using `fmt.Println`, which should be forbidden by the linter anyways. :)

File: cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go (hunk 3 above)
Comment (Ankitasw) [reply cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go:90]: I guess if no policy is passed, then we give empty line in console, we could actually make this intuitive and return a message saying "No policy doc provided" or something like that

Comment (Ankitasw) [reply cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go:90]: @Atharva-Shinde could you please take care of this ?

File: cmd/clusterawsadm/cloudformation/bootstrap/iam.go [hunk 4]
```diff
@@ -71,6 +71,18 @@ func (t Template) policyFunctionMap() map[PolicyName]func() *iamv1.PolicyDocumen
 	}
 }
 
+func (t Template) PrintPolicyDocs() error {
```
Comment (Skarlso) [inline cmd/clusterawsadm/cloudformation/bootstrap/iam.go:74]: I'm beginning to think this is some kind of attempt to undermine security because I still don't see anything changing here regardless of the force push.

**Comment by Atharva-Shinde** [comment]
Very disappointing @Skarlso. The reason for squashed commits is my attempt
to keep a clean commit history and address all the nits suggested,
additionally to my LFX mentorship feedback.

I am always open to feedbacks and constructive criticism but this comment
feels derogatory and insinuates my work ethics and integrity. I truly hope
this doesn't happen again to anyone else trying to contribute to the
wellbeing of the project. (or any project for that matter)

File: cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go (hunk 3 above)
Comment (Atharva-Shinde) [reply cmd/clusterawsadm/cmd/bootstrap/iam/iam_doc.go:90]: Thanks @Ankitasw I will make changes today itself :)

**Comment by Skarlso** [comment]
@Atharva-Shinde First of all, I voiced a concern, it's not towards you, it was towards the code.

Second... I told you a couple times now that I have an issue with that codesegment. Yet all you did was pushed some code somewhere, then pingged me to re-review meanwhile my comment was left untouched. Then pinged me again about the code, and I STILL don't see anything answered, commented or provided in any way as to explaining that segment.

I started to suspect that you either had some intent or didn't see my comment. But obviously, you see since you reacted to the last one. ;)

Now. I'm sorry if this offended you, but next time, please try to look at and answer possible review questions if you don't understand the question that is completely fine. :) You can ask for clarification. Ignoring things, however, will make me suspicious. :) For example, I thought that the change is in the controller side, that's why I frowned what the Println is doing there. Now I looked twice and saw it's in the CLI. That is my error. But it would have been completely fine if you would have answered that. :)

So let's try again. Hello. :) Could you please explain why that codesegment is there? And is the CLI possibly printing sensitive information that needs to be tempered? Also, since this is a CLI please provide at least an output of the running task how it looks like, how it's rendered.

**Comment by Atharva-Shinde** [comment]
Hey @Skarlso 
I respect the fact that being a maintainer/reviewer/owner can be volunteer work and indeed a tough job, but please be mindful of contributor's commitment factors as well. I myself am taking out time to learn and contribute on this which I am very passionate about but was busy these past few days! 
Therefore do you think it's fair that for not answering to your review statements (unintentionally or not) which you later found out was your mistake of not reviewing correctly should allow you to make such comment on me? Because otherwise I don't see a review statement that I have not addressed. Please refrain from commenting such remarks; comments like that might not be the best approach to confront out of all the other ways. 

And I deliberately say "comment on me" because 
> I started to suspect that you either had some intent or...

**Comment by Atharva-Shinde** [comment]
> So let's try again. Hello. :) Could you please explain why that code segment is there? And is the CLI possibly printing sensitive information that needs to be tempered? Also, since this is a CLI please provide at least an output of the running task how it looks like, and how it's rendered.

Perfect! 
Hey,

> Could you please explain why that code segment is there?

I assume by code segment you mean the PrintPolicyDocs function. 
PrintPolicyDocs is a function which just loops over the GetPolicyDocFromPolicyName function for each policyName and returns the policy document along with its policy name. PrintPolicyDocs returns nil if no error occurs when performing this operation.

> And is the CLI possibly printing sensitive information that needs to be tempered?

I am not sure :( but I think if the PrintPolicyDocs function just loops over GetPolicyDocFromPolicyName function which was was already in use, I don't think it prints any sensitive information.  
 
> Also, since this is a CLI please provide at least an output of the running task how it looks like, and how it's rendered.

Ack. Thank you I will keep this in mind :)

Output
```
$bin git:(printpolicy) ✗ ./clusterawsadm bootstrap iam print-policy

AWSIAMManagedPolicyControllers {
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "ec2:AttachNetworkInterface",
        "ec2:DetachNetworkInterface",
        "ec2:AllocateAddress",
        "ec2:AssignIpv6Addresses",
        "ec2:AssignPrivateIpAddresses",
        "ec2:UnassignPrivateIpAddresses",
        "ec2:AssociateRouteTable",
        "ec2:AttachInternetGateway",
        "ec2:AuthorizeSecurityGroupIngress",
        "ec2:CreateInternetGateway",
        "ec2:CreateEgressOnlyInternetGateway",
        "ec2:CreateNatGateway",
        "ec2:CreateNetworkInterface",
        "ec2:CreateRoute",
        "ec2:CreateRouteTable",
        "ec2:CreateSecurityGroup",
        "ec2:CreateSubnet",
        "ec2:CreateTags",
        "ec2:CreateVpc",
        "ec2:ModifyVpcAttribute",
        "ec2:DeleteInternetGateway",
        "ec2:DeleteEgressOnlyInternetGateway",
        "ec2:DeleteNatGateway",
        "ec2:DeleteRouteTable",
        "ec2:ReplaceRoute",
        "ec2:DeleteSecurityGroup",
        "ec2:DeleteSubnet",
        "ec2:DeleteTags",
        "ec2:DeleteVpc",
        "ec2:DescribeAccountAttributes",
        "ec2:DescribeAddresses",
        "ec2:DescribeAvailabilityZones",
        "ec2:DescribeInstances",
        "ec2:DescribeInstanceTypes",
        "ec2:DescribeInternetGateways",
        "ec2:DescribeEgressOnlyInternetGateways",
        "ec2:DescribeInstanceTypes",
        "ec2:DescribeImages",
        "ec2:DescribeNatGateways",
        "ec2:DescribeNetworkInterfaces",
        "ec2:DescribeNetworkInterfaceAttribute",
        "ec2:DescribeRouteTables",
        "ec2:DescribeSecurityGroups",
        "ec2:DescribeSubnets",
        "ec2:DescribeVpcs",
        "ec2:DescribeVpcAttribute",
        "ec2:DescribeVolumes",
        "ec2:DescribeTags",
        "ec2:DetachInternetGateway",
        "ec2:DisassociateRouteTable",
        "ec2:DisassociateAddress",
        "ec2:ModifyInstanceAttribute",
        "ec2:ModifyNetworkInterfaceAttribute",
        "ec2:ModifySubnetAttribute",
        "ec2:ReleaseAddress",
        "ec2:RevokeSecurityGroupIngress",
        "ec2:RunInstances",
        "ec2:TerminateInstances",
        "tag:GetResources",
        "elasticloadbalancing:AddTags",
        "elasticloadbalancing:CreateLoadBalancer",
        "elasticloadbalancing:ConfigureHealthCheck",
        "elasticloadbalancing:DeleteLoadBalancer",
        "elasticloadbalancing:DeleteTargetGroup",
        "elasticloadbalancing:DescribeLoadBalancers",
        "elasticloadbalancing:DescribeLoadBalancerAttributes",
        "elasticloadbalancing:DescribeTargetGroups",
        "elasticloadbalancing:ApplySecurityGroupsToLoadBalancer",
        "elasticloadbalancing:DescribeTags",
        "elasticloadbalancing:ModifyLoadBalancerAttributes",
        "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
        "elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
        "elasticloadbalancing:RemoveTags",
        "autoscaling:DescribeAutoScalingGroups",
        "autoscaling:DescribeInstanceRefreshes",
        "ec2:CreateLaunchTemplate",
        "ec2:CreateLaunchTemplateVersion",
        "ec2:DescribeLaunchTemplates",
        "ec2:DescribeLaunchTemplateVersions",
        "ec2:DeleteLaunchTemplate",
        "ec2:DeleteLaunchTemplateVersions",
        "ec2:DescribeKeyPairs"
      ],
      "Resource": [
        "*"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "autoscaling:CreateAutoScalingGroup",
        "autoscaling:UpdateAutoScalingGroup",
        "autoscaling:CreateOrUpdateTags",
        "autoscaling:StartInstanceRefresh",
        "autoscaling:DeleteAutoScalingGroup",
        "autoscaling:DeleteTags"
      ],
      "Resource": [
        "arn:*:autoscaling:*:*:autoScalingGroup:*:autoScalingGroupName/*"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:CreateServiceLinkedRole"
      ],
      "Resource": [
        "arn:*:iam::*:role/aws-service-role/autoscaling.amazonaws.com/AWSServiceRoleForAutoScaling"
      ],
      "Condition": {
        "StringLike": {
          "iam:AWSServiceName": "autoscaling.amazonaws.com"
        }
      }
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:CreateServiceLinkedRole"
      ],
      "Resource": [
        "arn:*:iam::*:role/aws-service-role/elasticloadbalancing.amazonaws.com/AWSServiceRoleForElasticLoadBalancing"
      ],
      "Condition": {
        "StringLike": {
          "iam:AWSServiceName": "elasticloadbalancing.amazonaws.com"
        }
      }
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:CreateServiceLinkedRole"
      ],
      "Resource": [
        "arn:*:iam::*:role/aws-service-role/spot.amazonaws.com/AWSServiceRoleForEC2Spot"
      ],
      "Condition": {
        "StringLike": {
          "iam:AWSServiceName": "spot.amazonaws.com"
        }
      }
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:PassRole"
      ],
      "Resource": [
        "arn:*:iam::*:role/*.cluster-api-provider-aws.sigs.k8s.io"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "secretsmanager:CreateSecret",
        "secretsmanager:DeleteSecret",
        "secretsmanager:TagResource"
      ],
      "Resource": [
        "arn:*:secretsmanager:*:*:secret:aws.cluster.x-k8s.io/*"
      ]
    }
  ]
}
AWSIAMManagedPolicyControllersEKS {
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "ssm:GetParameter"
      ],
      "Resource": [
        "arn:*:ssm:*:*:parameter/aws/service/eks/optimized-ami/*"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:CreateServiceLinkedRole"
      ],
      "Resource": [
        "arn:*:iam::*:role/aws-service-role/eks.amazonaws.com/AWSServiceRoleForAmazonEKS"
      ],
      "Condition": {
        "StringLike": {
          "iam:AWSServiceName": "eks.amazonaws.com"
        }
      }
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:CreateServiceLinkedRole"
      ],
      "Resource": [
        "arn:*:iam::*:role/aws-service-role/eks-nodegroup.amazonaws.com/AWSServiceRoleForAmazonEKSNodegroup"
      ],
      "Condition": {
        "StringLike": {
          "iam:AWSServiceName": "eks-nodegroup.amazonaws.com"
        }
      }
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:CreateServiceLinkedRole"
      ],
      "Resource": [
        "arn:aws:iam::*:role/aws-service-role/eks-fargate-pods.amazonaws.com/AWSServiceRoleForAmazonEKSForFargate"
      ],
      "Condition": {
        "StringLike": {
          "iam:AWSServiceName": "eks-fargate.amazonaws.com"
        }
      }
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:GetRole",
        "iam:ListAttachedRolePolicies"
      ],
      "Resource": [
        "arn:*:iam::*:role/*"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:GetPolicy"
      ],
      "Resource": [
        "arn:aws:iam::aws:policy/AmazonEKSClusterPolicy"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "eks:DescribeCluster",
        "eks:ListClusters",
        "eks:CreateCluster",
        "eks:TagResource",
        "eks:UpdateClusterVersion",
        "eks:DeleteCluster",
        "eks:UpdateClusterConfig",
        "eks:UntagResource",
        "eks:UpdateNodegroupVersion",
        "eks:DescribeNodegroup",
        "eks:DeleteNodegroup",
        "eks:UpdateNodegroupConfig",
        "eks:CreateNodegroup",
        "eks:AssociateEncryptionConfig",
        "eks:ListIdentityProviderConfigs",
        "eks:AssociateIdentityProviderConfig",
        "eks:DescribeIdentityProviderConfig",
        "eks:DisassociateIdentityProviderConfig"
      ],
      "Resource": [
        "arn:*:eks:*:*:cluster/*",
        "arn:*:eks:*:*:nodegroup/*/*/*"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "ec2:AssociateVpcCidrBlock",
        "ec2:DisassociateVpcCidrBlock",
        "eks:ListAddons",
        "eks:CreateAddon",
        "eks:DescribeAddonVersions",
        "eks:DescribeAddon",
        "eks:DeleteAddon",
        "eks:UpdateAddon",
        "eks:TagResource",
        "eks:DescribeFargateProfile",
        "eks:CreateFargateProfile",
        "eks:DeleteFargateProfile"
      ],
      "Resource": [
        "*"
      ]
    },
    {
      "Effect": "Allow",
      "Action": [
        "iam:PassRole"
      ],
      "Resource": [
        "*"
      ],
      "Condition": {
        "StringEquals": {
          "iam:PassedToService": "eks.amazonaws.com"
        }
      }
    },
    {
      "Effect": "Allow",
      "Action": [
        "kms:CreateGrant",
        "kms:DescribeKey"
      ],
      "Resource": [
        "*"
      ],
      "Condition": {
        "ForAnyValue:StringLike": {
          "kms:ResourceAliases": "alias/cluster-api-provider-aws-*"
        }
      }
    }
  ]
}
AWSIAMManagedPolicyCloudProviderControlPlane {
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "autoscaling:DescribeAutoScalingGroups",
        "autoscaling:DescribeLaunchConfigurations",
        "autoscaling:DescribeTags",
        "ec2:AssignIpv6Addresses",
        "ec2:DescribeInstances",
        "ec2:DescribeImages",
        "ec2:DescribeRegions",
        "ec2:DescribeRouteTables",
        "ec2:DescribeSecurityGroups",
        "ec2:DescribeSubnets",
        "ec2:DescribeVolumes",
        "ec2:CreateSecurityGroup",
        "ec2:CreateTags",
        "ec2:CreateVolume",
        "ec2:ModifyInstanceAttribute",
        "ec2:ModifyVolume",
        "ec2:AttachVolume",
        "ec2:AuthorizeSecurityGroupIngress",
        "ec2:CreateRoute",
        "ec2:DeleteRoute",
        "ec2:DeleteSecurityGroup",
        "ec2:DeleteVolume",
        "ec2:DetachVolume",
        "ec2:RevokeSecurityGroupIngress",
        "ec2:DescribeVpcs",
        "elasticloadbalancing:AddTags",
        "elasticloadbalancing:AttachLoadBalancerToSubnets",
        "elasticloadbalancing:ApplySecurityGroupsToLoadBalancer",
        "elasticloadbalancing:CreateLoadBalancer",
        "elasticloadbalancing:CreateLoadBalancerPolicy",
        "elasticloadbalancing:CreateLoadBalancerListeners",
        "elasticloadbalancing:ConfigureHealthCheck",
        "elasticloadbalancing:DeleteLoadBalancer",
        "elasticloadbalancing:DeleteLoadBalancerListeners",
        "elasticloadbalancing:DescribeLoadBalancers",
        "elasticloadbalancing:DescribeLoadBalancerAttributes",
        "elasticloadbalancing:DetachLoadBalancerFromSubnets",
        "elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
        "elasticloadbalancing:ModifyLoadBalancerAttributes",
        "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
        "elasticloadbalancing:SetLoadBalancerPoliciesForBackendServer",
        "elasticloadbalancing:CreateListener",
        "elasticloadbalancing:CreateTargetGroup",
        "elasticloadbalancing:DeleteListener",
        "elasticloadbalancing:DeleteTargetGroup",
        "elasticloadbalancing:DeregisterTargets",
        "elasticloadbalancing:DescribeListeners",
        "elasticloadbalancing:DescribeLoadBalancerPolicies",
        "elasticloadbalancing:DescribeTargetGroups",
        "elasticloadbalancing:DescribeTargetHealth",
        "elasticloadbalancing:ModifyListener",
        "elasticloadbalancing:ModifyTargetGroup",
        "elasticloadbalancing:RegisterTargets",
        "elasticloadbalancing:SetLoadBalancerPoliciesOfListener",
        "iam:CreateServiceLinkedRole",
        "kms:DescribeKey"
      ],
      "Resource": [
        "*"
      ]
    }
  ]
}
AWSIAMManagedPolicyCloudProviderNodes {
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "ec2:AssignIpv6Addresses",
        "ec2:DescribeInstances",
        "ec2:DescribeRegions",
        "ec2:CreateTags",
        "ec2:DescribeTags",
        "ec2:DescribeNetworkInterfaces",
        "ec2:DescribeInstanceTypes",
        "ecr:GetAuthorizationToken",
        "ecr:BatchCheckLayerAvailability",
        "ecr:GetDownloadUrlForLayer",
        "ecr:GetRepositoryPolicy",
        "ecr:DescribeRepositories",
        "ecr:ListImages",
        "ecr:BatchGetImage"
      ],
      "Resource": [
        "*"
      ]
    }
  ]
}
AWSEBSCSIPolicyController {
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "ec2:AttachVolume",
        "ec2:CreateSnapshot",
        "ec2:CreateTags",
        "ec2:CreateVolume",
        "ec2:DeleteSnapshot",
        "ec2:DeleteTags",
        "ec2:DeleteVolume",
        "ec2:DescribeAvailabilityZones",
        "ec2:DescribeInstances",
        "ec2:DescribeSnapshots",
        "ec2:DescribeTags",
        "ec2:DescribeVolumes",
        "ec2:DescribeVolumesModifications",
        "ec2:DetachVolume",
        "ec2:ModifyVolume"
      ],
      "Resource": [
        "*"
      ]
    }
  ]
}
```

**Comment by Skarlso** [comment]
I think you are overthinking this a bit. :) I merely stated a concern; whether it was valid or not is a different subject. You could have dispelled the concern with a single sentence. :) In the future, you'll be faced with much less reasonable people who always argue about nonsense. :)

Anyways, I apologise if my tone was upsetting. Here's to new beginnings. :)

Thanks for the output. I had a quick read, and it looks like other commands don't concern themselves with guarding against outputting secrets. I guess it is its purpose after all, so I think this is fine.

And you're right, if this was already possible anyways and this just extracts it as a simple separate command, it's probably fine as is. :)

So basically, the functionality was always there just individually and now you can print all of them.

/lgtm
/approve

**Comment by Skarlso** [comment]
Ah, I wonder. Do we need to update some kind of doc somewhere for the CLI help or something? 🤔

**Comment by Skarlso** [comment]
Right, would you mind updating the printPolicy command please and add an example where you call the thing without the`--document` flag here:

```go
func printPolicyCmd() *cobra.Command {
	newCmd := &cobra.Command{
		Use:   "print-policy",
		Short: "Generate and show an IAM policy",
		Long: cmd.LongDesc(`
			Generate and show an AWS Identity and Access Management (IAM) policy for
			Kubernetes Cluster API Provider AWS.
		`),
		Example: cmd.Examples(`
		# Print out the IAM policy for the Kubernetes Cluster API Provider AWS Controller.
		clusterawsadm bootstrap iam print-policy --document AWSIAMManagedPolicyControllers

		# Print out the IAM policy for the Kubernetes Cluster API Provider AWS Controller using a given configuration file.
		clusterawsadm bootstrap iam print-policy --document AWSIAMManagedPolicyControllers --config bootstrap_config.yaml		

		# Print out the IAM policy for the Kubernetes AWS Cloud Provider for the control plane.
		clusterawsadm bootstrap iam print-policy --document AWSIAMManagedPolicyCloudProviderControlPlane

		# Print out the IAM policy for the Kubernetes AWS Cloud Provider for all nodes.
		clusterawsadm bootstrap iam print-policy --document AWSIAMManagedPolicyCloudProviderNodes

		# Print out the IAM policy for the Kubernetes AWS EBS CSI Driver Controller.
		clusterawsadm bootstrap iam print-policy --document AWSEBSCSIPolicyController
		`),

```

Thanks. :)

**Comment by Atharva-Shinde** [comment]
Thanks for the review @Skarlso!

@Ankitasw does this address your comment or is there anything I could change to make this better.
